### ISSUE TITLE
[Docs] Waymo Dataset ground truth bin files

### ISSUE THREAD

**Issue by woo9904** [open-mmlab/mmdetection3d#2804]
### Branch

main branch  https://mmdetection3d.readthedocs.io/en/latest/

### 📚 The doc issue

Hello, 

According to this [link](https://github.com/open-mmlab/mmdetection3d/blob/main/docs/en/user_guides/dataset_prepare.md#waymo), I downloaded the Waymo open dataset V1.4.1 and its data split. 

But when I attempted to download the "ground truth .bin file", it seems that the file is missing from the [link](https://console.cloud.google.com/storage/browser/waymo_open_dataset_v_1_4_1/validation/ground_truth_objects;tab=objects?prefix=&forceOnObjectsSortingFiltering=false).

So Could you please re-upload the file? Or is it possible to use the ".bin" file that you have posted [here](https://mmdetection3d.readthedocs.io/en/latest/advanced_guides/datasets/waymo.html), even though it's a different version of the Waymo dataset? (The .bin file from this link looks like for V1.2.)

**Comment by ammaryasirnaich** [comment]
I cannot find the groundtruth(gt).bin file too, hoping to hear any solution from someone

**Comment by ammaryasirnaich** [comment]
I think i found it in v1.2 of waymo dataset [gt.bin](https://console.cloud.google.com/storage/browser/waymo_open_dataset_v_1_2_0/validation/ground_truth_objects?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false)

**Comment by woo9904** [comment]
@ammaryasirnaich Did you download waymo dataset 1.4 version?

**Comment by ammaryasirnaich** [comment]
> @ammaryasirnaich Did you download waymo dataset 1.4 version?

Yes i did and downloaded the gt.bin from v1.2

**Comment by woo9904** [comment]
> > @ammaryasirnaich Did you download waymo dataset 1.4 version?
> 
> Yes i did and downloaded the gt.bin from v1.2

If you can evaluate, can you tell me if it works well? I think that if the version is different, the evaluation will get weird results.

**Comment by woo9904** [comment]
So what we need to do right now is download the same version of dataset and gt.bin.

**Comment by ammaryasirnaich** [comment]
Yes, you could be right thats why i am getting strange results for pre-trained pointpiller model [2850](https://github.com/open-mmlab/mmdetection3d/issues/2850)
